### PR TITLE
docs(website): added documentation about applying a yaml manifest

### PR DESCRIPTION
**Pull request by shipsing**
### What does this PR do?
Adds documentation about applying a YAML manifest
### Screenshot / video of UI
PR preview: https://676265acc1162d8a04be928b--podman-desktop-pr.netlify.app/docs/kubernetes/applying-a-yaml-manifest

### What issues does this PR fix or reference?
https://github.com/podman-desktop/podman-desktop/issues/10096

### How to test this PR?

- [ ] Tests are covering the bug fix or the new feature

### CODE REVIEW

File: website/docs/kubernetes/applying-a-yaml-manifest.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+sidebar_position: 12
+title: Applying a YAML manifest
+description: How to apply a YAML manifest to create a kubernetes object
+keywords: [podman desktop, podman, objects, kubernetes]
+tags: [applying-a-yaml-manifest, creating-an-object]
+---
+
+# Applying a YAML manifest
+
+Based on your needs, you can apply a YAML manifest to create a Kubernetes object, such as `Node`, `Deployment`, `Service`, `PersistentVolumeClaim`, and others. For example, a Kubernetes deployment that requires sensitive data storage, you can apply a YAML manifest to create a `Secret` object.
```
Comment (cdrage) [inline website/docs/kubernetes/applying-a-yaml-manifest.md]:
```suggestion
You can deploy a YAML manifest to create a Kubernetes object, such as `Node`, `Deployment`, `Service`, `PersistentVolumeClaim`, and others. For example, a Kubernetes deployment that requires sensitive data storage, you can apply a YAML manifest to create a `Secret` object.
```

"Based on your needs" is kinda odd IMO

**Review by cdrage — CHANGES_REQUESTED** [review]

**Comment by cdrage** [comment]
Very small change, but otherwise LGTM!

**Review by Firewall — APPROVED** [review]
Awesome 👍

**Comment by shipsing** [comment]
@cdrage, Could you please verify the PR and confirm? Thanks

**Review by cdrage — APPROVED** [review]

**Pull request merged**